### ISSUE TITLE
Minimum zone ventilation efficiency

### ISSUE THREAD

**Issue by lymereJ** [NREL/OpenStudio#3898]
Enhancement Request
--------------
The SDK allows users to set/get the zone air distribution effectiveness in heating/cooling mode via `DesignSpecificationZoneAirDistribution` and `SizingZone` but not the minimum zone ventilation efficiency, which is a valid input in EnergyPlus. It would great if the later could be added.

**Comment by jmarrec** [comment]
Confirmed it's missing in OpenStudio, and easy to add.

```
OS:DesignSpecification:ZoneAirDistribution,
   [...]
   N3;  \field Zone Secondary Recirculation Fraction
        \type real
        \default 0.0
        \minimum 0.0
        \units dimensionless
```

replace with 

```
OS:DesignSpecification:ZoneAirDistribution,
   [...]
   N3,  \field Zone Secondary Recirculation Fraction
        \type real
        \default 0.0
        \minimum 0.0
        \units dimensionless
   N4;  \field Minimum Zone Ventilation Efficiency
        \type real
        \default 0.0
        \minimum 0.0
        \maximum 1.0
        \units dimensionless
```

**Comment by jmarrec** [comment]
EUH, actually, I seems that while the object appears to be wrapped in the model API (`DesignSpecificationZoneAirDistribution` has a class, and registered constructors etc), it's not being used during forward translation at all. Instead the `OS:Sizing:Zone` is the one that adds some of the fields directly.

**Comment by lymereJ** [comment]
Could this be included in the next OpenStudio release?

**Comment by tijcolem** [comment]
@jmarrec Can we get this in for 3.0.0?  Run regression tests to make sure no major changes?

**Comment by jmarrec** [comment]
Its possible, but not as straightforward as I had thought initially since the existing class is actually not used. Requires modifying the model API to make use of it, or strapping the new field onto the SizingZone object like it is now (less work for sure). 

@kbenne thoughts? I would assume put it on OS:Sizing:Zone?

**Comment by tijcolem** [comment]
closed and merged in #3915
